### ISSUE TITLE
Add Google Doc

### ISSUE THREAD

**Issue by agardnerIT** [open-feature/protocol#10]
This Google doc should probably be accessible from this repo (the readme)?

I'll leave it to those more familiar with OFREP to decide if / where it goes.

https://docs.google.com/document/d/12WRZLMDYSNYYDMwowCC-f6tosyloEdmBBwAuAAXFoew/edit#heading=h.cyy3attbcv66

**Comment by thomaspoignant** [comment]
@agardnerIT good idea but I am not sure this is up-to-date enough to be linked in the readme.
But I think most of the useful information from this doc should be added to the repo.
